data/data-pipeline/data_pipeline/etl/sources/census_decennial/etl.py

```diff
@@ -210,6 +287,23 @@ def __init__(self):
             self.EMPLOYMENT_MALE_UNEMPLOYED_FIELD: self.EMPLOYMENT_MALE_UNEMPLOYED_FIELD,
             self.EMPLOYMENT_FEMALE_IN_LABOR_FORCE_FIELD: self.EMPLOYMENT_FEMALE_IN_LABOR_FORCE_FIELD,
             self.EMPLOYMENT_FEMALE_UNEMPLOYED_FIELD: self.EMPLOYMENT_FEMALE_UNEMPLOYED_FIELD,
+            self.TOTAL_RACE_POPULATION_FIELD: self.TOTAL_RACE_POPULATION_FIELD_NAME,
+            self.TOTAL_RACE_POPULATION_VI_FIELD: self.TOTAL_RACE_POPULATION_FIELD_NAME,
+            self.AMERICAN_INDIAN_VI_FIELD: self.AMERICAN_INDIAN_FIELD_NAME,
```

Comment:
We'll also need a row above this that is: `self.AMERICAN_INDIAN_FIELD: self.AMERICAN_INDIAN_FIELD_NAME,` right?

Reply:
Sadly, it doesn't look like they have data on American Indian race/ethnicity for AS/GU/MI that I could find. I should add a comment to show it's intentional though

Reply:
Oh ok. Thanks!

Reply:
Comment added in b741001be6e1918273ddbc77f4431d507011d5df